framework/scripts/wazuh_server.py

```diff
@@ -354,7 +354,7 @@ async def monitor_server_daemons(loop: asyncio.BaseEventLoop, server_process: ps
     """
     comms_api_config = CentralizedConfig.get_comms_api_config()
     process_children = {
-        MANAGEMENT_API_DAEMON_NAME[:15]: 5,
+        MANAGEMENT_API_DAEMON_NAME[:15]: 4,
```

Comment:
We should update this function's unit test

Reply:
Fixed at 50834aa71e13ff9348a14ba8208703ed96c5d17e